### PR TITLE
Try to avoid some covariant issues

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Review by jordisala1991 — DISMISSED** [review]
If fixes the issues in: https://github.com/sonata-project/SonataMediaBundle/pull/2007

**Comment by VincentLanglet** [comment]
I'm waiting the opinion of phpstan/psalm maintainers

**Comment by jordisala1991** [comment]
So what is better , to just supress the issue on media-bundle or to add this code? (as mentioned in https://github.com/phpstan/phpstan/issues/5254#issuecomment-875768048)

**Comment by VincentLanglet** [comment]
It's described as "unexpected" but not really wrong https://github.com/phpstan/phpstan/issues/5254#issuecomment-875894397
I have currently no response from psalm.

But I would say it's the best way we have to solve these issues.

Ignoring the error in the media bundle will only solve it for us, and every user doing something similar will have the issue, so I prefer solving it on SonataAdmin.

**Review by dmaicher — APPROVED** [review]

File: src/Action/RetrieveAutocompleteItemsAction.php [hunk 1]
```diff
@@ -196,6 +196,9 @@ public function __invoke(Request $request): JsonResponse
      * @param AdminInterface<object> $admin
```
Comment (franmomu) [inline src/Action/RetrieveAutocompleteItemsAction.php:196]: Does it make sense these anymore?

**Review by franmomu — DISMISSED** [review]

File: src/Action/RetrieveAutocompleteItemsAction.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Action/RetrieveAutocompleteItemsAction.php]: Fixed

**Review by franmomu — APPROVED** [review]

**Pull request merged**